src/gui/src/clockWidget.cpp

```diff
@@ -1043,16 +1065,43 @@ ClockNodeGraphicsViewItem* ClockTreeView::addBufferToScene(
   const qreal delay_y
       = convertDelayToY(output_pin.delay) - convertDelayToY(input_pin.delay);
 
-  ClockBufferNodeGraphicsViewItem* node
-      = new ClockBufferNodeGraphicsViewItem(input_term, output_term, delay_y);
-
-  auto* lib_port = network->libertyPort(output_pin.pin);
+  ClockNodeGraphicsViewItem* node = nullptr;
+  bool is_clockgate = false;
+  bool is_inverter_buffer = false;
+  sta::LibertyPort* lib_port = network->libertyPort(output_pin.pin);
   if (lib_port != nullptr) {
-    auto function = lib_port->function();
-    if (function && function->op() == sta::FuncExpr::op_not) {
-      node->setIsInverter(true);
+    sta::LibertyCell* cell = lib_port->libertyCell();
+    if (cell->isClockGate()) {
+      is_clockgate = true;
+    } else {
+      if (cell->isBuffer() || cell->isInverter()) {
+        is_inverter_buffer = true;
+      }
+    }
+  } else {
+    is_inverter_buffer = true;
```

Comment:
If there is no liberty perhaps we should treat this as 'other'?

Reply:
We could, but I would assumed that they are likely buffers and inverters, so thats why I left them as the default without the liberty files.

Reply:
CTS will error if you give it a cell w/o a Liberty to use.  So any of these would have to be non-CTS inserted cells.

Reply:
okay, I'll switch the default to false